CONTRIBUTING.md

```diff
@@ -96,3 +96,31 @@ The workflow for contributions is fairly simple:
 3. make whatever changes you want to contribute.
 4. ensure your contribution does not introduce linting issues or breaks the tests by linting and testing the code.
 5. make a pull request with an adequate description.
+
+
+## Releasing `djangorestframework-stubs`
+
+1. Open a pull request that updates `setup.py` (anyone can open this PR, not just maintainers):
+
+   - Increase `version=` value within `setup(...)`. Version number `major.minor.patch` is formed as follows:
+
+     `major.minor` version must match newest supported `djangorestframework` release.
+
+     `patch` is sequentially increasing for each stubs release. Reset to `0` if `major.minor` was updated.
+
+   - Update `django-stubs>=` dependency to point to latest `django-stubs` release.
+   - Use pull request title "Version x.y.z release" by convention.
+
+2. Ensure the CI succeeds. A maintainer must merge this PR. If it's just a verison bump, no need to wait
+   for a second maintainer's approval.
+
+3. [Create a new GitHub release](https://github.com/typeddjango/djangorestframework-stubs/releases/new)
+
+   - Under "Choose a tag" enter the new version number. Do NOT use `v` prefix.
+   - Click "Generate release notes".
+   - Delete all release notes lines containing `by @pre-commit-ci` or `by @dependabot`, as these are
+     irrelevant for our users.
```

Comment:
It's an additional manual step, but IMO release notes are too noisy if these are included.